Casks/wireshark.rb

```
@@ -18,15 +18,20 @@
   uninstall_preflight do
     set_ownership '/Library/Application Support/Wireshark'
 
-    if File.read('/etc/group').match?(%r{^access_bpf})
+    if system('/usr/bin/dscl . -read /Groups/access_bpf &> /dev/null')
```

Use `system_command`, as below.

---

Sorry, I'm not much familiar with ruby nor homebrew files.
I've tried to use `system_command`, but if the group is not preset, the whole uninstall process fails with an error, instead of just skipping the command inside the `if` statement.
Probably it's just because of my inability, but how should I re-write the statement?